Aquarius/Src/Aquarius/Renderer/RenderingContext.cpp

```diff
@@ -0,0 +1,45 @@
+#include "RenderingContext.h"
+#include <Aquarius/Core/Log.h>
+
+
+namespace Aquarius {
+
+    RenderingContext::RenderingContext(GLFWwindow* window)
+    :   m_Window(window)
+    {}
+
+    RenderingContext::ContextPtr RenderingContext::Create(GLFWwindow *window)
+    {
+        return std::make_unique<RenderingContext>(window);
+    }
+
+    void RenderingContext::Initialize()
+    {
+        // Use GLAD to load opengl function pointers
+        glfwMakeContextCurrent(m_Window);
+        if (!gladLoadGLLoader((GLADloadproc)glfwGetProcAddress))
+        {
+            AQ_CORE_FATAL("Failed to initialize GLAD" );
+
+            // Set GLFW error callback
+            glfwSetErrorCallback([](int error_code, const char* description)
+            {
+                AQ_CORE_ERROR("GLFW Error Occurred:");
+                AQ_CORE_ERROR(description);
+            });
```

Comment:
Switch the order of these so that the error callback code actually gets reached before program termination.
P.S. nice lambda